wallets/react-wallet-v2/src/views/SessionProposalModal.tsx

```diff
@@ -3,36 +3,23 @@ import ProposalSelectSection from '@/components/ProposalSelectSection'
 import RequestModalContainer from '@/components/RequestModalContainer'
 import SessionProposalChainCard from '@/components/SessionProposalChainCard'
 import ModalStore from '@/store/ModalStore'
-import { cosmosAddresses } from '@/utils/CosmosWalletUtil'
 import { eip155Addresses } from '@/utils/EIP155WalletUtil'
-import { polkadotAddresses } from '@/utils/PolkadotWalletUtil'
-import { elrondAddresses } from '@/utils/ElrondWalletUtil'
-import { tronAddresses } from '@/utils/TronWalletUtil'
-import { tezosAddresses } from '@/utils/TezosWalletUtil'
-import {
-  isCosmosChain,
-  isEIP155Chain,
-  isSolanaChain,
-  isPolkadotChain,
-  isNearChain,
-  isElrondChain,
-  isTronChain,
-  isTezosChain
-} from '@/utils/HelperUtil'
-import { solanaAddresses } from '@/utils/SolanaWalletUtil'
+import { isEIP155Chain } from '@/utils/HelperUtil'
 import { signClient } from '@/utils/WalletConnectUtil'
 import { Button, Divider, Modal, Text } from '@nextui-org/react'
 import { SessionTypes } from '@walletconnect/types'
 import { getSdkError } from '@walletconnect/utils'
-import { Fragment, useState } from 'react'
-import { nearAddresses } from '@/utils/NearWalletUtil'
+import { Fragment, useEffect, useState } from 'react'
 
 export default function SessionProposalModal() {
   const [selectedAccounts, setSelectedAccounts] = useState<Record<string, string[]>>({})
   const hasSelected = Object.keys(selectedAccounts).length
 
   // Get proposal data and wallet address from store
   const proposal = ModalStore.state.data?.proposal
+  useEffect(() => {
+    console.log('selectedAccounts', selectedAccounts)
```

Comment:
Wanted or debug only?

Reply:
Now that the examples have been stable for quite some time, there's no need for the logs